packages/components/src/listbox/tests/chromatic/Listbox.chroma.jsx

```diff
@@ -0,0 +1,672 @@
+import { Avatar } from "@components/avatar";
+import { PlaceholderMajorIcon, CrossMajorIcon, IconList, LightbulbMajorIcon, NotificationMajorIcon } from "@components/icons";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong